python_cli/liquidai_cli/commands/model.py

```diff
@@ -5,11 +5,91 @@
 from typing import Optional, Dict, List, cast
 from liquidai_cli.utils.docker import DockerHelper
 from liquidai_cli.utils.config import load_config
+from typing_extensions import Annotated
 
 app = typer.Typer(help="Manage ML models")
 docker_helper = DockerHelper()
 
 
+@app.command(name="run-model-image")
+def run_model_image(
+    name: str = typer.Option(..., "--name", help="Name for the model"),
+    model_image: str = typer.Option(..., "--image", help="Model image name"),
+    port: Annotated[int, typer.Option("--port", help="Port to expose locally")] = 9000,
+    gpu: Annotated[str, typer.Option("--gpu", help="Specific GPU index to use")] = "all",
```

Comment:
This comment is for future.

We currently use `all` GPUs by default. But after a second thought, I think this default is not ideal. The launch commands are not for multi-GPUs. So the default should be one GPU instead. But this does not need to be addressed in this PR.